docker-compose-monitor.yml

```diff
@@ -0,0 +1,109 @@
+#
```

Comment:
@agolovnev-godeltech i suggest to fix `docker-compose.yml` which already exists rather than create new one. I don't think we are going to have any issues during re-base of our changes on latest master of parent repository. Edit of existing file makes clear what line where changed. At the moment whole file is considered to be `new` and it will be difficult to notice than only few line were changed.